test/com/collections/MethodsTest.java

```diff
@@ -103,25 +103,64 @@ void reverseArrayList(){
         list2.add(employee2);
         list2.add(employee1);
 
-        assertEquals(methods.reverseArrayList(list1),list2);
+        assertEquals(methods.reverseArrayList(list1), list2);
     }
 
-    @Test void checkIfAnagram(){
-        assertTrue(methods.checkIfAnagram("dupa","upad"));
-        assertTrue(methods.checkIfAnagram("moszna","mszona"));
-        assertTrue(methods.checkIfAnagram("sieckarnia","kaseciarni"));
+    @Test
+    void checkIfAnagram() {
+        assertTrue(methods.checkIfAnagram("dupa", "upad"));
+        assertTrue(methods.checkIfAnagram("moszna", "mszona"));
+        assertTrue(methods.checkIfAnagram("sieckarnia", "kaseciarni"));
 
-        assertFalse(methods.checkIfAnagram("dupa","zupa"));
-        assertFalse(methods.checkIfAnagram("drzewo","JavaCollections"));
+        assertFalse(methods.checkIfAnagram("dupa", "zupa"));
+        assertFalse(methods.checkIfAnagram("drzewo", "JavaCollections"));
 
     }
 
-    @Test void canBePalindrome(){
+    @Test
+    void canBePalindrome() {
         assertTrue(methods.canBePalindrome("kajak"));
         assertTrue(methods.canBePalindrome("kkaaj"));
         assertTrue(methods.canBePalindrome("kkaa"));
         assertFalse(methods.canBePalindrome("abcd"));
         assertFalse(methods.canBePalindrome("kajakk"));
     }
 
+    @Test
+    void sumTwoIntArrays() {
```

Comment:
It will be nice to add more cases (arrays with different length, one empty array etc). You can create each case as a separate test.

You can also prepare one test and create **data provider** to prepare different data sets for test. But here separate tests will be ok. Data provider can be good candidate for the next task.